src/ligo.ml

```diff
@@ -143,6 +143,10 @@ let int_from_literal s =
   with exc -> failwith ("Ligo.int_from_literal: " ^ Printexc.to_string exc)
 
 let int_from_int64 = Z.of_int64
+let nat_from_int64 (t: Int64.t) =
+  let r = Z.of_int64 t in
+  assert (r > Z.zero);
```

Comment:
Probably changing `random_ptr` is the best of the two options.

Reply:
I think changing `nat_from_int64` is better, since `0` is a natural number.  It might be better for `random_ptr` to produce a `0`, so it can surface some bugs if we're handling `null_ptr` differently.

Reply:
Can't argue with that; having `random_ptr` generate pointers in the entire domain is probably for the best, let's leave this as is :+1: